python/swiftnav/nav_msg.pxd

```diff
@@ -21,21 +22,22 @@ cdef extern from "libswiftnav/nav_msg.h":
     BIT_POLARITY_UNKNOWN
 
   ctypedef struct nav_msg_t:
```

Comment:
Yeah, you also need to declare: `BIT_LENGTH_MAX`. the semicolons are also redundant.